packages/wdio-browserstack-service/src/util.ts

```diff
@@ -1120,3 +1133,26 @@ export async function pushDataToQueue(data: UploadType, requestQueueHandler: Req
 }
 
 export const sleep = (ms = 100) => new Promise((resolve) => setTimeout(resolve, ms))
+
+export async function uploadLogs(user: string | undefined, key: string | undefined, clientBuildUuid: string) {
```

Comment:
Is the request actually succeeding if no `user` and `key` is provided? if you only allow `undefined` so it doesn't throw a type issue I recommend to have something like:

```ts
if (user && key) {
    uploadLogs(...)
}
```